packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -335,6 +407,52 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
               />
             </section>
           )}
+
+          {config.showServiceQueueFields && (
+            <>
+              <section className={styles.section}>
+                <div className={styles.sectionTitle}>{t('service', 'Service')}</div>
+                <Select
+                  labelText={t('selectService', 'Select a service')}
+                  id="service"
+                  invalidText="Required"
+                  value={service}
+                  onChange={(event) => setSelectedService(event.target.value)}
+                >
+                  {!service ? <SelectItem text={t('chooseService', 'Select a service')} value="" /> : null}
+                  {allServices?.length > 0 &&
+                    allServices.map((service) => (
+                      <SelectItem key={service.uuid} text={service.display} value={service.uuid}>
+                        {service.display}
+                      </SelectItem>
+                    ))}
+                </Select>
+              </section>
+
+              <section className={styles.section}>
+                <div className={styles.sectionTitle}>{t('priority', 'Priority')}</div>
+                <ContentSwitcher
+                  size="sm"
+                  selectionMode="manual"
+                  onChange={(event) => {
+                    setPriority(event.name as any);
+                  }}
+                >
+                  {priorities?.length > 0 ? (
+                    priorities.map(({ uuid, display }) => {
+                      return <Switch name={uuid} text={display} value={uuid} index={uuid} />;
+                    })
+                  ) : (
+                    <Switch
+                      name={t('noPriorityFound', 'No priority found')}
+                      text={t('noPriorityFound', 'No priority found')}
+                      value={null}
+                    />
```

Comment:
This should go into an error / empty state. Having this as part of the content switcher could be misleading. I'd recommend having a look at [this implementation](https://github.com/openmrs/openmrs-esm-patient-chart/blob/6dcb94693937127edf9f8f1cd0d1a32355d5d768/packages/esm-patient-programs-app/src/programs/programs-form.component.tsx#L185) for inspiration.